[2025-08-08T10:37:57.386+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-08T10:37:57.581+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_data_pipeline.fetch_and_store_stock_data manual__2025-08-08T10:37:55.639178+00:00 [queued]>
[2025-08-08T10:37:57.619+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_data_pipeline.fetch_and_store_stock_data manual__2025-08-08T10:37:55.639178+00:00 [queued]>
[2025-08-08T10:37:57.622+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-08-08T10:37:57.678+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_and_store_stock_data> on 2025-08-08 10:37:55.639178+00:00
[2025-08-08T10:37:57.689+0000] {standard_task_runner.py:63} INFO - Started process 267 to run task
[2025-08-08T10:37:57.717+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'stock_data_pipeline', 'fetch_and_store_stock_data', 'manual__2025-08-08T10:37:55.639178+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/stock_dag.py', '--cfg-path', '/tmp/tmp4mvw4uo4']
[2025-08-08T10:37:57.774+0000] {standard_task_runner.py:91} INFO - Job 8: Subtask fetch_and_store_stock_data
[2025-08-08T10:37:57.937+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-08T10:37:58.111+0000] {task_command.py:426} INFO - Running <TaskInstance: stock_data_pipeline.fetch_and_store_stock_data manual__2025-08-08T10:37:55.639178+00:00 [running]> on host e50c8dd86a0f
[2025-08-08T10:37:58.462+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='stock_data_pipeline' AIRFLOW_CTX_TASK_ID='fetch_and_store_stock_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-08T10:37:55.639178+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-08T10:37:55.639178+00:00'
[2025-08-08T10:37:58.466+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-08T10:37:58.519+0000] {fetch_stock_data.py:76} ERROR - [Attempt 1] Network/API error: Invalid URL 'None': No scheme supplied. Perhaps you meant https://None?
[2025-08-08T10:38:03.522+0000] {fetch_stock_data.py:76} ERROR - [Attempt 2] Network/API error: Invalid URL 'None': No scheme supplied. Perhaps you meant https://None?
[2025-08-08T10:38:08.522+0000] {fetch_stock_data.py:76} ERROR - [Attempt 3] Network/API error: Invalid URL 'None': No scheme supplied. Perhaps you meant https://None?
[2025-08-08T10:38:08.522+0000] {fetch_stock_data.py:83} CRITICAL - Failed to fetch data after retries.
[2025-08-08T10:38:08.523+0000] {fetch_stock_data.py:92} WARNING - No stock data to insert.
[2025-08-08T10:38:08.524+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-08T10:38:08.525+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-08T10:38:08.542+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=stock_data_pipeline, task_id=fetch_and_store_stock_data, run_id=manual__2025-08-08T10:37:55.639178+00:00, execution_date=20250808T103755, start_date=20250808T103757, end_date=20250808T103808
[2025-08-08T10:38:08.593+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-08T10:38:08.614+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-08T10:38:08.617+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
